<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimonGame</class>
 <widget class="QMainWindow" name="SimonGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimonGame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    this-&gt;setStyleSheet(&quot;QMainWindow {background-color: qlineargradient( x1:0 y1:0, x2:1 y2:1, stop:0 rgb(255,148,130), stop:1 rgb(125,119,255));}&quot;);
    ui-&gt;startButton-&gt;setStyleSheet(&quot;QPushButton{background: rgb(234,242,245); border-radius: 15px;}&quot;);
    ui-&gt;blueButton-&gt;setStyleSheet( QString(&quot;QPushButton {background-color: rgb(0,0,255);font: bold 20px;color: rgb(0,0,0);} QPushButton:pressed {background-color: rgb(52,153,255);}&quot;));
    ui-&gt;redButton-&gt;setStyleSheet( QString(&quot;QPushButton {background-color: rgb(240,11,81);font: bold 20px; color: rgb(0,0,0);} QPushButton:pressed {background-color: rgb(255,150,150);}&quot;));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>45</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gameTitleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simon Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="progressBarLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Progress:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>463</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="scoreNameLabel">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>500</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="highScoreNameLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High Score:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="highScoreLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
